File: 1-two-sum/two-sum.go
func twoSum(nums []int, target int) []int {
    for index, value := range nums{
        for index2 := index + 1; index2 < len(nums); index2++ {
            if value + nums[index2] == target {
                return []int{index, index2}
            }
        }
    }
    return nil
}

File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        tot = {}
        for i in range(len(nums)):
            if nums[i] in tot:
                return([tot[nums[i]],i])
            else:
                tot[target - nums[i]] = i

        